queries.yaml

```diff
@@ -53,7 +53,11 @@ layers:
     template: landuse.jinja2
     start_zoom: 4
     geometry_types: [Polygon, MultiPolygon]
+<<<<<<< HEAD
```

Comment:
D'oh! Thanks for catching that. I thought `git` had a lint for that... obviously not. Sorry. Fixed in 162f1bd.